### PR TITLE
[12.0][MIG] base_iso3166

### PR DESCRIPTION
**Pull request by aitorbouzas**
No major changes.

### CODE REVIEW

File: base_iso3166/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+# Copyright 2017  Creu Blanca <www.creublanca.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "ISO 3166",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "author": "Tecnativa, "
               "Creu Blanca, "
+              "AdaptiveCity, "
```
Comment (pedrobaeza) [inline base_iso3166/__manifest__.py]: I think not significant changes for being co-author.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: base_iso3166/__manifest__.py (hunk 1 above)
Comment (emagdalenaC2i) [reply base_iso3166/__manifest__.py]: I think the same when I see the README.rst of project_description... 
Seems to me that the new auto-generated README.rst will show AdaptiveCity as Author anyway.
See project_description and https://github.com/OCA/project/pull/439
Edit: No, sorry. My mistake.
It was there before in v11.
I don't remember to change that. But probably someone tell me to do that XD

Comment (aitorbouzas) [reply base_iso3166/__manifest__.py]: But I added some tests :(

Comment (aitorbouzas) [reply base_iso3166/__manifest__.py]: @emagdalenaC2i does AdaptiveCity appears somewhere else? I think I did not contribute in the project repository.

**Comment by aitorbouzas** [comment]
@pedrobaeza Removed authority and squashed commits.

File: base_iso3166/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply base_iso3166/__manifest__.py]: Well, Aitor, you know that the effort for doing that little test has not been very high. But don't worry, you will soon appear in several modules with the pending refactorings we have in mind and don't forget that you always appear in contributors section.

File: base_iso3166/README.rst [hunk 2]
```diff
@@ -35,37 +57,43 @@ Known issues / Roadmap
...
 
 Credits
 =======
 
+Authors
+~~~~~~~
+
+* Tecnativa
+* Creu Blanca
+* AdaptiveCity
```
Comment (pedrobaeza) [inline base_iso3166/README.rst]: Rebuild the README

Comment (aitorbouzas) [reply base_iso3166/README.rst]: Done

**Review by pedrobaeza — APPROVED** [review]

**Review by astirpe — APPROVED** [review]

**Pull request merged**